importing/service.go

```diff
@@ -77,33 +49,40 @@ func (is *importingService) getDataFromFile(fileName string) []*Spending {
 			s.Type = INCOME
 		}
 		if !s.Type.IsValid() {
-			panic("Invalid spending type")
+			return nil, fmt.Errorf("invalid spending type: %v", s.Type)
 		}
 	}
 
-	return spending
+	return spending, nil
 }
 
+// Todo: consider to return (n, error). With n is the total of written spendings.
 func (is *importingService) writeSpending(spending []*Spending) {
 	for _, s := range spending {
-		p := is.spendingToPoint(s)
-		err := is.influx.WritePoint(p)
+		p, err := is.spendingToPoint(s)
 		if err != nil {
-			panic(err)
+			log.Printf("convert spending to point: %v\n", err)
+			continue
```

Comment:
I think it should be best if we can revert all the data we imported so far if we got problem here 🤔